workspace-loader/src/index.ts

```diff
@@ -322,7 +306,7 @@ export class WorkspaceLoader {
             masterApi.subscribeWorkspaceStatus(this.workspace.id,
                 (message: any) => {
                     if (message.error) {
-                        this.loader.error(message.error);
+                        reject(`Failed to run the workspace: "${message.error}"`);
                     } else if (message.status === 'RUNNING') {
                         resolve();
```

Comment:
Can we, please, also check "runtime" is there (not null), when we have RUNNING status. If not - reject the promise. 
There is usecase, when user tries to open shared workspace, where he has no permissions on runtime.

Reply:
sure, I'll add a fixup

Reply:
More details here https://github.com/eclipse/che/issues/10687
Maybe we can also handle second use case here when POST /workspace/WS_ID/runtime is failed

Reply:
Than POST is handled here
https://github.com/eclipse/che/blob/389e259d7672fe57300f10ad19657076ea7f19f9/workspace-loader/src/index.ts#L211-L212

Reply:
Cool. I missed it because PR title and even description tell nothing about this change.

Looks like we will be able to close that issue after merge. I can recheck after merge.

Reply:
It was another PR https://github.com/eclipse/che/pull/10642 which is already merged.

Reply:
I see, thanks. 
I did find the given lines in master and you pointed commit from this PR, it's why I thought that it is introduced in this PR =)